### ISSUE TITLE
ROS2

### ISSUE THREAD

**Issue by YuexinMu** [MISTLab/ROSBuzz#9]
Author Will you produce a version of ROS2？

**Comment by beltrame** [comment]
Yes, that is the plan. We are doing a lot of changes at the moment, eventually we will have a new release. @vivek-shankar

**Comment by vivek-shankar** [comment]
I appreciate your interest in ROSBuzz.  We will be working on a ROS2 version in the coming months; you can expect to see a new release soon.

**Comment by YuexinMu** [comment]
OK, Thanks.
